spec/workers/vbms/submit_dependents_pdf_job_spec.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe VBMS::SubmitDependentsPDFJob do
+  let(:invalid_dependency_claim) { create(:dependency_claim_no_vet_information) }
+  let(:dependency_claim) { create(:dependency_claim) }
+
+  let(:vet_info) do
+    {
+      'veteran_information' => {
+        'birth_date' => '1809-02-12',
+        'full_name' => {
+          'first' => 'WESLEY', 'last' => 'FORD', 'middle' => nil
+        },
+        'ssn' => '796043735',
+        'va_file_number' => '796043735'
+      }
+    }
+  end
```

Comment:
If you defined this as a model class, you could then easily define a factory for it.